src/Sulu/Bundle/SearchBundle/Tests/Functional/Controller/SearchControllerTest.php

```diff
@@ -41,6 +41,17 @@ class SearchControllerTest extends SuluTestCase
      */
     private $user;
 
+    public function setUp()
+    {
+        parent::setUp();
+        $this->purgeDatabase();
+        $this->client = $this->createAuthenticatedClient();
+        $this->searchManager = $this->client->getContainer()->get('massive_search.search_manager');
```

Comment:
Shouldn't this also be `$this->getContainer()`? Maybe there should also be a section in the documentation about this.